pcs/lib/commands/resource.py

```diff
@@ -1698,15 +1698,23 @@ def move_autoclean(
     if env.report_processor.report_list(report_list).has_errors:
         raise LibraryError()
 
+    cluster_state=env.get_cluster_state()
```

Comment:
```suggestion
    cluster_state = env.get_cluster_state()
```

We use https://github.com/psf/black for python code formatting. Make sure that you run `make check` before committing changes. To enable all development related checks (e.g. pylint, mypy, black and typos checks) run `./configure` with option `--enable-dev-tests`. See [CONTRIBUTING.md](https://github.com/ClusterLabs/pcs/blob/main/CONTRIBUTING.md) for more details.

You can use `make black` to automatically format all python code.